pep-0463.txt

```diff
@@ -3,14 +3,42 @@ Title: Exception-catching expressions
 Version: $Revision$
 Last-Modified: $Date$
 Author: Chris Angelico <rosuav@gmail.com>
-Status: Draft
+Status: Rejected
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 15-Feb-2014
 Python-Version: 3.5
 Post-History: 20-Feb-2014, 16-Feb-2014
 
 
+Rejection Notice
```

Comment:
Where does this rejection notice come from? The text is written in the first-person but I don't know who it is that's speaking.